### PR TITLE
EZEE-2716: Richtext block: tooltip for links can render in way that part of it won't be visible

### PR DESCRIPTION
**Pull request by czechmarcin**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-2716
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by dew326** [comment]
Seems like this could fix other issue: https://jira.ez.no/browse/EZEE-2701 or the fix can be easily change to fix both.

**Comment by dew326** [comment]
From the ticket it looks like this is a partial fix. I underdstand the ticket in way that issue is with position of the toolbar. It should be moved to the left to fit the visible window, but probably @mateuszbieniek should say what is expected by the customer.

**Comment by mateuszbieniek** [comment]
I think this approach is more of a workaround than real fix, as it enables the user to use the feature, but does not resolve all UI issues.

On the other hand, the Customer expects that all toolbar buttons are visible and clickable, which this PR provides and preparing more sofisticated solution will be a lot more complicated and time consuming.

To sum it up: +1 from me.

**Comment by dew326** [comment]
Ok. So we need to check if this fix the other issue with buttons (probably customer will complain on it but he didn't see it yet) at the bottom and we are good to go.

**Comment by czechmarcin** [comment]
Used common class `alloy-editor-visible` instead of `ae-toolbar-styles`. Works on both - anchor and custom tag.

**Comment by lserwatka** [comment]
@dew326 can we proceed with QA here?

File: src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css [hunk 1]
```diff
@@ -1044,3 +1044,7 @@
   font-size: 14px;
   line-height: 1;
 }
+
+.alloy-editor-visible {
```
Comment (dew326) [inline src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css]: I don't know this class. Where it's added?

Comment (czechmarcin) [reply src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css]: This class is added for modals with `custom tag` and `anchor`.

Comment (czechmarcin) [reply src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css]: Thanks to this change, the `custom tag`:
- visibility of `custom tag` will have a bigger priority than buttons bar
- if the `custom tag` element exceeds the height limit, overflow appears

Comment (dew326) [reply src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css]: This problem probably will be for other toolbars as well. What do you think to add this to the parent `ae-toolbars` and kill bug once for all?

Comment (czechmarcin) [reply src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css]: Adding `z-index: 1;` to `.ae-toolbars` won't solve the problem.
Child needs this attribute.
Screenshot 2019-03-14 at 10 04 19
Screenshot 2019-03-14 at 10 04 09

**Comment by czechmarcin** [comment]
@barbaragr ATM I cannot reproduce your case. Despite that, my proposition is to apply these commits because:
- this change makes the modal visible instead of hidden under some layers (top and bottom)
- even if we don't have full modal, overflow appears and we can scroll to see the box

**Comment by micszo** [comment]
It works with tag v2.4.2, it doesn't with stable 2.4. The issue is ezplatform-admin-ui-assets version. Tag has 3.2.0, stable branch installs 3.1.0. It's caused by composer.json entry "~3.1.0@dev" (vs. "^3.1.0@dev"). Thanks @dew326 !

**Comment by lserwatka** [comment]
@micszo feel free to make a PR to update assets package version.

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by czechmarcin** [comment]
done

**Pull request merged**